academy/4-my-own-chain/can-play.md

```diff
@@ -19,7 +19,7 @@ Make sure you have all you need before proceeding:
 
 A player sends a `MsgPlayMove` when [making a move](./play-game.md). This message can succeed or fail for several reasons. One error situation is when the message represents an invalid move.
 
-Players should be able to make sure that a move is valid before burning gas. To add this functionality, you need to create a way for the player to call the [`Move`](https://github.com/batkinson/checkers-go/blob/a09daeb/checkers/checkers.go#L274) function without changing the game's state. Use a query because they are evaluated in memory and do not commit anything permanently to storage.
+Players should be able to make sure that a move is valid before burning gas. To add this functionality, you need to create a way for the player to call the [`Move`](https://github.com/batkinson/checkers-go/blob/a09daeb/checkers/checkers.go#L274) function without changing the game's state. You use a query because they are evaluated in memory and do not commit anything permanently to storage to achieve this.
```

Comment:
The "this" that is "achieved" is what is said in the previous sentence, in particular "without changing the game's state". After [moving it from the beginning of the sentence to its end](https://github.com/cosmos/sdk-tutorials/commit/0d92c160f34a7af6c028595dc77d19fe87fd6acc#diff-bc420a8829c2734eeae1aa8581254ba6dfb1fa8d6265e3964078df31fabc4a39R22), "this" is very far from what it represents and it is no longer clear what it points back at.

Reply:
"to achieve this" now relocated to the beginning of the sentence to improve connection to subject concept: _...without changing the game's state. **You use a query to achieve this,** because..._